src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -248,9 +254,52 @@ export function QueryPageContainerComponent(): JSX.Element {
             setSubFilters(adaptedSubFilters);
             setUniqueDataValues(uniqueValuesObject);
             setTotalNrOfIsol(totalNrOfIsolates);
+            setDataIsMounted(true);
         }
     };
 
+    const fetchNewFilterValues = async (): Promise<void> => {
+        setConditionalValuesAreLoading(true);
+        const newUniqueValues = _.cloneDeep(uniqueDataValues);
+        const filterStatusList: number[] = [];
+        const getConditionalFilterValues = filter.displayedFilters.map(
+            async (mainFilter) => {
+                const selectedFilter = filter.selectedFilter[mainFilter];
+                if (_.isEmpty(selectedFilter)) {

```

Comment:
Yes, you are right. 
I'll have to discuss it with Dominic. 
Because when I fetch the conditional values for a selected filter, I only get back the selected filter value itself. 
For your example: 

- organism A: year 2017
- organism B: year 2018
- organism C: year 2017

URL: /filterconfig/organism/organism=organismA
API returns only "OrganismA" as filter value and not other organisms (Organism C) that would be possible for 2017.

Reply:
Yes, you'r right. 
It's done.